pkg/option/features.go

```diff
@@ -1,18 +1,18 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright Authors of Cilium
 
-package identitycachecell
+package option
 
-import "github.com/cilium/cilium/pkg/option"
-
-func netPolicySystemIsEnabled(cfg *option.DaemonConfig) bool {
+// NetPolicySystemIsEnabled returns true if the network policy enforcement
+// system is enabled for K8s, Cilium and Cilium Clusterwide network policies.
+func NetPolicySystemIsEnabled(cfg *DaemonConfig) bool {
```

Comment:
While at it, this could be simplified to use `cmp.Or`:

```go
func NetPolicySystemIsEnabled(cfg *DaemonConfig) bool {
	return cmp.Or(
		cfg.EnablePolicy != NeverEnforce,
		cfg.EnableK8sNetworkPolicy,
		cfg.EnableCiliumNetworkPolicy,
		cfg.EnableCiliumClusterwideNetworkPolicy,
		!cfg.DisableCiliumEndpointCRD,
		cfg.IdentityAllocationMode != IdentityAllocationModeCRD,
	)
```

Reply:
Done.